scripts/sync_branding.sh

```diff
@@ -26,7 +26,13 @@ valora_branding_sha=89f86a0ce54d7d06afe914ea6f4a057affe09024
 if [[ "$branding" == "valora" ]]; then
   # prevents git from asking credentials
   export GIT_TERMINAL_PROMPT=0
-  if [[ ! -e branding/valora ]] && ! git clone git@github.com:valora-inc/valora-app-branding.git branding/valora ; then
+  remote_path=valora-inc/valora-app-branding.git
+  local_dir=branding/valora
+  if [[ ! -e $local_dir ]] && 
+    ( 
+      ! git clone git@github.com:$remote_path $local_dir &&
+      ! git clone https://github.com/$remote_path $local_dir 
```

Comment:
nit: to avoid surprises with variables expansion
```suggestion
  if [[ ! -e "$local_dir" ]] && 
    ( 
      ! git clone "git@github.com:$remote_path" "$local_dir" &&
      ! git clone "https://github.com/$remote_path" "$local_dir" 
```

Reply:
Thanks!